r/environment.yml

```diff
@@ -3,14 +3,15 @@ channels:
   - conda-forge
 dependencies:
   # base
-  - python=3.9
-  - pangeo-notebook=2023.02.11
+  - python=3.10
+  - pangeo-notebook=2023.07.21
   - nb_conda_kernels
   - jupyter-rsession-proxy
   # 2/24/23: Added jsonschema-with-format-nongpl to address a deployment error
   - jsonschema-with-format-nongpl
   - gh
   - nano
+  - mamba
```

Comment:
We won't save much but we can remove mamba and configure libmamba in the condarc.

Reply:
I want my `mamba repoquery whoneeds why-the-heck-is-this-so-large`